bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "localforage-cordovasqlitedriver",
   "version": "1.4.1",
   "main": [
-    "src/localforage-cordovasqlitedriver.js"
+    "dist/localforage-cordovasqlitedriver.js"
```

Comment:
You are totally right. Nice catch :+1: I probably forgot to change it while migrating to rollup.